web/tests/EducationBenchmarking.Web.E2ETests/Pages/CurriculumFinancialPlanning/ICFPHelpPage.cs

```diff
@@ -0,0 +1,40 @@
+﻿using EducationBenchmarking.Web.E2ETests.Helpers;
+using EducationBenchmarking.Web.E2ETests.Hooks;
+using EducationBenchmarking.Web.E2ETests.TestSupport;
+using FluentAssertions;
+using Microsoft.Playwright;
+
+namespace EducationBenchmarking.Web.E2ETests.Pages.CurriculumFinancialPlanning;
+
+public class ICFPHelpPage(PageHook page)
+{
+    private readonly IPage _page = page.Current;
+    private ILocator PageH1Heading => _page.Locator("h1");
+    //todo update the backLink selector when it has been added
+    private ILocator BackLink => _page.Locator("#link-back");
```

Comment:
Should this PR stay in 'draft' until the link is available? The test run currently fails due to this.

Reply:
Amm since we don't have these set up in pipeline as of yet so it won't block anything even if it fails. The dev work needs to be done and i believe its standard to name the back link with this id. either way it would be quick tweak once the dev work is done.

Reply:
It looks like the selector will be `.govuk-back-link`

Reply:
Thank you checking and confirming. Updated it now.